elliottlib/bzutil.py

```diff
@@ -621,6 +626,9 @@ def get_tracker_bugs(self, bug_ids: List, strict: bool = False, verbose: bool =
     def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
         return [b for b in self.get_bugs(bug_ids, permissive=not strict, verbose=verbose) if b.is_flaw_bug()]
 
+    def filter_attached_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
+        return [b for b in bug_ids if JiraIssue(b).all_advisory_ids != []]
```

Comment:
Think this is more pythonic than testing equalisty with empty list: `if not JiraIssue(b).all_advisory_ids`